src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -190,4 +190,16 @@ end
 
     return nothing
 end
+
+@inline function get_boundary_outer_state(u_inner, cache, t,
```

Comment:
It seems that this function is only implemented for `boundary_condition::BoundaryConditionDirichlet` and will return a "no method matching" error if the minmax limiter is used with another boundary condition. Is that the case?
Anyway, it would be good to add a docstring here indicating the arguments and what this function does.

Reply:
Yes, that's the case right now. This is because I only added a non-periodic elixir with `BoundaryConditionDircihlet` yet.

If I remember correctly, we additionally implemented this function for `boundary_condition_slip_wall` and the exilir-depending boundary conditions `BoundaryConditionCharacteristic` and `boundary_condition_mixed_dirichlet_wall`. Right now, it only would make sense to add `boundary_condition_slip_wall` here, since the others aren't even implemented yet.
Only the other hand, maybe it would be useful to remove all non-periodic elixirs and implementations first for this PR and add this stuff later. What do you think?

Reply:
I now decided to keep the non-periodic implementation for `BoundaryConditionDirichlet` in here. Nevertheless, I remove some parameter of this routine since they are only required later for other boundary conditions. And I added the docstring.